firmware/CMakeLists.txt

```diff
@@ -94,7 +107,7 @@ function(cube_mx_code_generation
     add_custom_command(
         OUTPUT ${IOC_FILE}.md5
         ${LOG4J_PROPERTIES}
-        COMMAND python3
+        COMMAND ${XVFB_PREFIX} python3
```

Comment:
I assume that the build system error `xvfb-run: error: Xvfb failed to start` is coming from here.

Reply:
Yeah I'm trying to fix this now

Reply:
it doesn't happen when building locally